packages/figma-utils/src/variables/generate.ts

```diff
@@ -65,16 +76,24 @@ export const generateAsSCSS = (data: ParsedVariable, options?: BaseGenerateOptio
  *
  * @returns File content of the .json file
  */
-export const generateAsJSON = (data: ParsedVariable): string => {
-  const variables = structuredClone(data.variables);
+export const generateAsJSON = (data: ParsedVariable, dataDarkTheme?: ParsedVariable): string => {
```

Comment:
For JSON I don't see the need to support generating dark variables since its exactly the same as calling this function for light and then for dark only.

So basically we can keep the previous behavior and just call the function twice with different data passed to it